backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -125,4 +119,45 @@ protected RestAssuredResponse login(final String nickname) {
 
         return null;
     }
+
+    protected RestAssuredResponse requestCreateTeam(final String title, final String token,
+                                                    final int interviewerNumber, final List<Long> participantIds) {
+        return requestCreateTeam(title, token, interviewerNumber, participantIds, TEAM_START_TIME);
+    }
+
+    protected RestAssuredResponse requestCreateTeam(final String title, final String token, final int interviewerNumber,
+                                                    final List<Long> participantIds, final LocalDateTime startTime) {
+        final ParticipantIdsDto participantIdsDto = new ParticipantIdsDto(participantIds);
+        final TeamWriteDto request = new TeamWriteDto(title, title + "place", interviewerNumber, startTime,
+                participantIdsDto);
+
+        return post("/api/teams", token, request);
+    }
+
+    protected RestAssuredResponse requestCreateLevellog(final String content, final String teamId, final String token) {
+        final LevellogWriteDto request = LevellogWriteDto.from(content);
+
+        return post("/api/teams/" + teamId + "/levellogs", token, request);
+    }
+
+    protected RestAssuredResponse requestCreateFeedback(final String content, final String levellogId,
+                                                        final String token) {
+        final FeedbackWriteDto request = FeedbackWriteDto.from(
+                "study " + content, "speak " + content, "etc " + content);
+
+        return post("/api/levellogs/" + levellogId + "/feedbacks", token, request);
+    }
+
+    protected RestAssuredResponse requestCreatePreQuestion(final String content, final String levellogId,
+                                                           final String token) {
+        final PreQuestionDto request = PreQuestionDto.from(content);
+
+        return post("/api/levellogs/" + levellogId + "/pre-questions/", token, request);
+    }
+
+    protected RestAssuredResponse requestCreateInterviewQuestion(final String content, final String levellogId,
+                                                                 final String token) {
+        return post("/api/levellogs/" + levellogId + "/interview-questions", token,
+                InterviewQuestionDto.from(content));
```

Comment:
```suggestion
        final InterviewQuestionDto request = InterviewQuestionDto.from(content);

        return post("/api/levellogs/" + levellogId + "/interview-questions", token, request);
```
다른 메서드와 달리 여기서만 인라인으로 값이 들어가고있어요! 다른 메서드처럼 나눠주면 더 코드읽기가 쉬울 것 같은데 어떠세요?